plexapi/media.py

```diff
@@ -264,50 +307,71 @@ class AudioStream(MediaPartStream):
             bitrate (int): Audio bitrate (ex: 448).
             bitrateMode (str): Bitrate mode (ex: cbr).
             channels (int): number of channels in this stream (ex: 6).
-            dialogNorm (int): Unknown (ex: -27).
             duration (int): Duration of audio stream in milliseconds.
             samplingRate (int): Sampling rate (ex: xxx)
-            title (str): Title of this audio stream.
     """
     TAG = 'Stream'
     STREAMTYPE = 2
 
     def _loadData(self, data):
         """ Load attribute values from Plex XML response. """
         super(AudioStream, self)._loadData(data)
+        self.albumGain = cast(float, data.attrib.get('albumGain'))
+        self.albumPeak = cast(float, data.attrib.get('albumPeak'))
+        self.albumRange = cast(float, data.attrib.get('albumRange'))
         self.audioChannelLayout = data.attrib.get('audioChannelLayout')
         self.bitDepth = cast(int, data.attrib.get('bitDepth'))
-        self.bitrate = cast(int, data.attrib.get('bitrate'))
         self.bitrateMode = data.attrib.get('bitrateMode')
         self.channels = cast(int, data.attrib.get('channels'))
-        self.dialogNorm = cast(int, data.attrib.get('dialogNorm'))
         self.duration = cast(int, data.attrib.get('duration'))
+        self.endRamp = data.attrib.get('endRamp')
+        self.gain = cast(float, data.attrib.get('gain'))
+        self.loudness = cast(float, data.attrib.get('loudness'))
+        self.lra = cast(float, data.attrib.get('lra'))
+        self.peak = cast(float, data.attrib.get('peak'))
+        self.profile = data.attrib.get('profile')
         self.samplingRate = cast(int, data.attrib.get('samplingRate'))
-        self.title = data.attrib.get('title')
+        self.startRamp = data.attrib.get('startRamp')
 
```

Comment:
Add `streamIdentifier`.
```py
self.streamIdentifier = cast(int, data.attrib.get('streamIdentifier'))
```

Reply:
so both video and audio streams? If so, then it should be in MediaPartStream.

Reply:
Only `VideoStream` and `AudioStream`, so not factored out to `MediaPartStream`.